### PR TITLE
Support any type of token (erc20, erc721, erc1155)

### PR DESCRIPTION
**Pull request by atharvadeosthale**
Now supports any type of tokens (ERC20, ERC721, ERC1155)! Just replace the contract address in `yourDetails.js` file.

### CODE REVIEW

**Comment by jarrodwatts** [comment]
Could you give the readme a quick update? I'll also change the name of this template to token-gated-website

**Comment by atharvadeosthale** [comment]
On it!

File: README.md [hunk 1]
```diff
@@ -1,25 +1,25 @@
 # NFT Gated Website
 
-This project demonstrates how you can restrict content on your website to only those users who own an NFT from your collection.
+This project demonstrates how you can restrict content on your website to only those users who own a token of your choice (any ERC-20, ERC-721 or ERC-1155).
 
-We use an [Edition Drop](https://portal.thirdweb.com/pre-built-contracts/edition-drop) contract to enable users to claim one of the NFTs, and serve users the restricted content if they have at least one of the NFTs claimed.
+You can use any token you like, although we will recommend you to use either [Token Drop](https://thirdweb.com/thirdweb.eth/DropERC20), [NFT Drop](https://thirdweb.com/thirdweb.eth/DropERC721) or [Edition Drop](https://thirdweb.com/thirdweb.eth/DropERC1155). That's because we will have a button to claim these test tokens for testing this template. You can also use any other token you like regardless of them being a drop token.
```
Comment (jarrodwatts) [inline README.md:5]: Let's not recommend the prebuilts but let's explain if your contract is not a thirdweb one you need to go through the import flow

File: README.md [hunk 2]
```diff
@@ -1,25 +1,25 @@
...
 
 Create a project using this example:
 
 ```bash
-npx thirdweb create --template nft-gated-website
+npx thirdweb create --template token-gated-website
 ```
 
-- Create an [Edition Drop](https://thirdweb.com/contracts/new/pre-built/drop/edition-drop) contract using the dashboard.
+- Deploy a contract from the links provided above.
```
Comment (jarrodwatts) [inline README.md:20]: Deploy or import your contract

File: README.md (hunk 1 above)
Comment (atharvadeosthale) [reply README.md:5]: Sure, although the buttons to claim the drops for testing only work with drop contracts- wouldn't it be confusing to test initially?

**Pull request merged**